lightning-invoice/src/payment.rs

```diff
@@ -17,7 +17,7 @@ use lightning::ln::channelmanager::RecipientOnionFields;
 use lightning::routing::router::{PaymentParameters, RouteParameters};
 
 /// Builds the necessary parameters to pay or pre-flight probe the given zero-amount
```

Comment:
```suggestion
/// Builds the necessary parameters to pay or pre-flight probe the given
```
and then wrap so lines are 100chars.